src/main/java/yoda/QuestList.java

```diff
@@ -0,0 +1,62 @@
+package yoda;
+
+/**
+ * Represents a list of all existing quests.
+ * @author Tonishka Singh
+ */
+
+import java.util.ArrayList;
+
+public class QuestList {
+    private ArrayList<Quest> questList;
+
+    public QuestList(ArrayList<Quest> questList) {
+        this.questList = questList;
+    }
+
+    public void addQuest(Quest q) {
+        questList.add(q);
+    }
+
+    public Quest getQuest(int questID) {
+        return questList.get(questID);
+    }
+
+    public Quest deleteQuest(int questID) {
+        Quest q = this.getQuest(questID);
+        questList.remove(questID);
+        return q;
+    }
+
+    public int numQuests() {
+        return questList.size();
+    }
+
+    public ArrayList<Quest> find(String keyword) {
+        ArrayList<Quest> result = new ArrayList<>();
```

Comment:
Perhaps the name could be changed to something like results or matches? List objects should be written as plural.